### ISSUE TITLE
Lists: specify in the succeeded action, not the pending?

### ISSUE THREAD

**Issue by jcrben** [jmeas/redux-resource#255]
Apologies if I missed something obvious:

The latest guide to lists (https://github.com/jmeas/redux-resource/blob/82c9477ca89996ad1a1e88a88328898a20a7879f/docs/guides/lists.md) shows the `list` key on both the pending and succeeded action. However, I found that only the key in the succeeded action seemed to matter. The code seems to support that: the reducer-generator doesn't seem to do anything with the `list` key while the https://github.com/jmeas/redux-resource/blob/82c9477ca89996ad1a1e88a88328898a20a7879f/packages/redux-resource/src/utils/cru-reducer-helper.js (which seems like it could be renamed to reflect its focus on success) uses it.

Couple approaches that cross my mind: 
* Maybe it applies if provided in either the pending or success?
* Remove it from the pending action example and update the docs to note that "You only need to specify a list for create or read actions *in their success action*".

**Comment by jmeas** [comment]
:wave: Hey again @jcrben !

> However, I found that only the key in the succeeded action seemed to matter.

Yup, this is correct.

> Maybe it applies if provided in either the pending or success?

Currently, it will only apply when supplied along with the success action.

> Remove it from the pending action example and update the docs to note that "You only need to specify a list for create or read actions in their success action".

:+1: This sounds like a good solution to me!

---

Thanks for catching this and opening the issue, @jcrben !

**Comment by jcrben** [comment]
Made some edits, feel free to edit it up at your leisure. Figured I'd reuse the existing callout note.

redux-loop seems to be working OK so far altho I'm just noodling around. I was thinking I would just use https://github.com/jmeas/redux-resource/issues/186 to start for abstracting over

what node and npm version are you using? when I tried `npm run bootstrap` running node@7.9.0 and npm@5.5.1 my computer froze up for a few minutes until I managed to kill the process

I like to put a `.node-version` in my projects (nodenv respects it, nvm will soon) - may look at running my npm installs thru a docker executable container to lock things down further since npm has been flakey lately
